source/app/pages/documents/view.js

```diff
@@ -418,45 +432,46 @@ function Document({ currentPageNumber, dispatch, id, document, pageTitle, search
               />
 
               <EntitiesCheckbox
-                 entities={docData.entities}
-                 pageCount={pageCount}
-                 currentPageNumber={currentPageNumber}
-                 showRedaction={track === 'redaction'}
-                 onHighlight={highlightEntities}
-                 onSwitchPage={switchPage}
-                 onRedact={redactEntityMatches}
-                 onRedactAll={redactAllValues}
-                 onDownload={downloadKV}
-                 visible={tab === 'entities'}
-                 comprehendService={COMPREHEND_SERVICE}
-                 onDownloadPrimary = {downloadEntities}
-                 onDownloadSecondary = {null}
-                 document = {document}
+                entities={docData.entities}
+                pageCount={pageCount}
+                currentPageNumber={currentPageNumber}
+                showRedaction={track === 'redaction'}
+                onHighlight={highlightEntities}
+                onSwitchPage={switchPage}
+                onRedact={redactEntityMatches}
+                onRedactAll={redactAllValues}
+                onDownload={downloadKV}
+                visible={tab === 'entities'}
+                comprehendService={COMPREHEND_SERVICE}
+                onDownloadPrimary = {downloadEntities}
+                onDownloadSecondary = {null}
+                document = {document}
               />
 
-              <EntitiesCheckbox
-                 entities={docData.medicalEntities}
-                 pageCount={pageCount}
-                 currentPageNumber={currentPageNumber}
-                 showRedaction={track === 'redaction'}
-                 onHighlight={highlightEntities}
-                 onSwitchPage={switchPage}
-                 onRedact={redactEntityMatches}
-                 onRedactAll={redactAllValues}
-                 onDownloadPrimary={downloadMedicalEntities}
-                 onDownloadSecondary = {downloadMedicalICD10Ontologies}
-                 visible={tab === 'medical_entities'}
-                 comprehendService={COMPREHEND_MEDICAL_SERVICE}
-                 document = {document}
-              />
+              { ENABLE_COMPREHEND_MEDICAL?
```

Comment:
`{ condition && <Component /> }` is preferable to `{ condition ? <Component /> : null }`

Reply:
updated